recipes/ngs-disambiguate/meta.yaml

```diff
@@ -1,23 +1,27 @@
+{% set version = "1.0.0" %}
+
 package:
   name: ngs-disambiguate
-  version: "2016.11.10"
```

Comment:
@johanneskoester the new version will be `older` than the old one right? So we want to retire the older version?

Reply:
I just checked and the release was on the same day as the tag. Nevertheless, people might already pin against the "date"-version. Unsure how to solve this...

Reply:
We could label it as `broken`? And people who have pinned it needs to enable this label?

Reply:
Yes, you are right. Thinking about it, that makes also sense because the version is actually broken, due to the missing pinning. Let's do that.

Reply:
+1 It also doesn't look like that version was an official one anyway: https://github.com/AstraZeneca-NGS/disambiguate/blame/v1.0.0/setup.py